packages/framework-integration-tests/package.json

```diff
@@ -111,11 +112,12 @@
     "integration/aws-end-to-end": "echo 'Skipping integration tests.' # TODO 'Enable it again when AWS Provider will be upgrade it' TS_NODE_PROJECT=\"./tsconfig.integration.json\" TESTED_PROVIDER=AWS AWS_SDK_LOAD_CONFIG=true BOOSTER_ENV=production mocha --forbid-only --exit --config \"integration/provider-unaware/end-to-end/.mocharc.yml\" \"integration/provider-unaware/end-to-end/**/*.integration.ts\"",
     "integration/aws-load": "echo 'Skipping integration tests.' # TS_NODE_PROJECT=\"./tsconfig.integration.json\" TESTED_PROVIDER=AWS AWS_SDK_LOAD_CONFIG=true BOOSTER_ENV=production mocha --forbid-only --exit --config \"integration/provider-unaware/load/.mocharc.yml\" \"integration/provider-unaware/load/**/*.load.ts\"",
     "integration/aws-nuke": "TS_NODE_PROJECT=\"./tsconfig.integration.json\" AWS_SDK_LOAD_CONFIG=true BOOSTER_ENV=production mocha --forbid-only --exit --config \"integration/provider-specific/aws/nuke/.mocharc.yml\" \"integration/provider-specific/aws/nuke/**/*.integration.ts\"",
-    "integration/local": "TS_NODE_PROJECT=\"./tsconfig.integration.json\" npm run integration/local-start && npm run integration/local-end-to-end && npm run integration/local-stop",
+    "integration/local": "concurrently \"npm run integration/local-start\" \"npm run integration/local-delayed-end-to-end\"",
```

Comment:
🫨 What is this black magic of "concurrently"? I normally do "script_1; script_2"... Is this word an oficial operator in bash?

Reply:
[concurrently](https://github.com/open-cli-tools/concurrently) fixes a very annoying problem that occurred when running the integration tests on the local provider. Ideally, it was as simple as running `integration/local-start`, `integration/local-end-to-end`, and `integration/local-stop` in sequence because when `integration/local-start` finishes, it leaves the server running in the background until `integration/local-stop` is executed. However, for some unknown reason, the server was being stopped/killed right in the middle of `integration/local-end-to-end`. I observed that when keeping `integration/local-start` open (i.e., removing the `--exit` flag from the `mocha` command) and then running `integration/local-end-to-end` in another terminal window the end-to-end tests finished just fine. So what concurrently does is "simulate" having those two terminals open. The `sleep` is added to `integration/local-end-to-end` so that it waits enough time for the server to start since concurrently spawns the commands it receives simultaneously.

Reply:
All right. Thanks for the explanation. Wondering why that started happening now.